keanu-project/src/main/java/io/improbable/keanu/network/BayesianNetwork.java

```diff
@@ -63,6 +81,22 @@ public Vertex getVertexByLabel(VertexLabel label) {
         return vertices;
     }
 
+    public int getVertexCount() {
+        return vertexCount;
+    }
+
+    public Integer getVertexDegree(Vertex vertex) {
+        return vertexDegrees.get(vertex.getId());
+    }
+
+    public Integer getVertexDegree(VertexId id) {
+        return vertexDegrees.get(id);
```

Comment:
For this one and the previous method - would we want to return null or throw an exception if a vertex that is not a part of the network is passed in?